cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -57,3 +65,40 @@ func startMizarPodController(ctx *ControllerContext, grpcHost string) (http.Hand
 	).Run(mizarPodControllerWorkerCount, ctx.Stop)
 	return nil, true, nil
 }
+
+func startMizarServiceController(ctx *ControllerContext, grpcHost string) (http.Handler, bool, error) {
+	controllerName := "mizar-service-controller"
+	klog.V(2).Infof("Starting %v", controllerName)
+
+	svcKubeconfigs := ctx.ClientBuilder.ConfigOrDie(controllerName)
+	svcKubeClient := clientset.NewForConfigOrDie(svcKubeconfigs)
+	networkClient := arktos.NewForConfigOrDie(svcKubeconfigs)
+	informerFactory := externalversions.NewSharedInformerFactory(networkClient, 10*time.Minute)
```

Comment:
I have a bit concern here.  Network is a CRD, which may not exist in the system. Is it better to check the existence of this CRD before moving on.

It does not block my approval, though. Please check with h-w-chen for the best solution.